I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 9, in script call
    call ch2_main from _call_ch2_main
  File "game/script-ch2.rpy", line 403, in script
    br "ej bro,har du prøvet at tag dem på timon"
Exception: Sayer 'br' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 9, in script call
    call ch2_main from _call_ch2_main
  File "game/script-ch2.rpy", line 403, in script
    br "ej bro,har du prøvet at tag dem på timon"
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 2579, in execute
    Say.execute(self)
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 581, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\Jacob\Desktop\renpy-8.3.4-sdk\renpy\ast.py", line 470, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'br' is not defined.

Windows-10-10.0.19045 AMD64
Ren'Py 8.3.4.24120703
A Week Of School 2 1.0.0
Tue Dec 17 19:25:40 2024
